fhir_transformer/src/transform/transformer.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+from fhir_transformer.src.config.logger import create_logger
+from fhir_transformer.src.transform.fhir import create_instance
+from fhir_transformer.src.helper import get_topic_name
+
+logger = create_logger("transformer")
+
+
+class Transformer:
+    def __init__(self, producer):
+        self.producer = producer
+
+    def transform(self, row, analysis):
+        return create_instance(row, analysis)
+
+    def process_event(self, msg):
```

Comment:
I think I would separate the process event function (used for kafka consumer) and the class Transformer (to create the fhir resource). Having the producer as an input of this class does not make a lot of sense to me